glam/api/models.py

```diff
@@ -267,3 +267,19 @@ class Meta:
                 fields=["channel", "build_id"],
             )
         ]
+
+
+class UsageInstrumentation(models.Model):
+    ACTION_PROBE_SEARCH = 'PROBE_SEARCH'
```

Comment:
In other files you've corrected quoting to be double quotes, which is what `black` prefers. Before the final commit it might be worthwhile to run all the files through `black` and `isort` if you want. I believe I did this but it looks like I never set up any automation or CI checks for it.

Reply:
fixed!